coordinator/src/services/translation-service.ts

```diff
@@ -258,6 +256,30 @@ function convertProduct(medicationCodeableConcept: fhir.CodeableConcept) {
     return new prescriptions.Product(manufacturedProduct);
 }
 
+export function convertCourseOfTherapyType(medicationRequests: Array<fhir.MedicationRequest>): prescriptions.PrescriptionTreatmentType {
+    const courseOfTherapyTypeCodes = new Set(
```

Comment:
Could adapt getCodeableConceptCodingForSystem to do this (takes array of CodeableConcepts and a code system, currently returns the single result - could instead make a Set to return the single **unique** result)

Reply:
Or could just get the value from the first MedicationRequest in the bundle as we do for various other fields (perhaps not a good idea)